tests/mbt/model/ccv_model.qnt

```diff
@@ -225,8 +328,8 @@ module ccv_model {
                     if (commonPackets.size() == 0) {
                         true // they don't share any packets, so nothing to check
                     } else {
-                        val newestCommonPacket = commonPackets.maxBy(packet => packet.sendingTime)
-                        val oldestCommonPacket = commonPackets.minBy(packet => packet.sendingTime)
+                        val newestCommonPacket = commonPackets.latest()
```

Comment:
nit: maybe `latest()` could be renamed to `newest` and similarly `earliest()` to `oldest` because, for example, saying `latest` might confuse someone to think of the `oldest`.